File: 11.27+开发心得/STEncryptUtils(AES和Base64加密)/STEncryptUtils/ViewController.m
//
//  ViewController.m
//  STEncryptUtils
//
//  Created by yls on 14-1-15.
//  Copyright (c) 2014年 yls. All rights reserved.
//

#import "ViewController.h"
#import "STEncryptUtils.h"

// 秘钥
#define Key @"stlwtr"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    
    NSString *pwd = @"123456";
    // 加密
    NSString *enPwd = [pwd encryptWithKey:Key];
    
    NSLog(@"%@", enPwd);
    
    // 解密
    NSString *dePwd = [enPwd decryptWithKey:Key];
    
    NSLog(@"%@", dePwd);
    
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: 2017.11.28+心得/featurestudydemo/CallKit/callkit 2/VoIP-Demo-swift/VoIP-Demo/ViewController.swift
//
//  ViewController.swift
//  VoIP-Demo
//
//  Created by Stefan Natchev on 2/5/15.
//  Copyright (c) 2015 ZeroPush. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var tokenLabel: UILabel!
    @IBOutlet weak var payloadLabel: UILabel!

    @IBAction func requestButtonPressed(_ sender: UIButton) {
        let appDelegate = UIApplication.shared.delegate as! AppDelegate
        appDelegate.registerVoipNotifications()
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

